From b25e7dab52bb4cf41c4547d6dc602d126272b824 Mon Sep 17 00:00:00 2001
From: Sean Mackrory <sean@cloudera.com>
Date: Fri, 17 Feb 2017 16:19:15 -0700
Subject: [PATCH 2269/2748] CLOUDERA-BUILD. Include TestDynamoDBMetadataStore
 in compilation, but do not run.

The test was previously unable to compile due to the AWS SDK version,
however it is still unable to run due to other runtime dependency
conflicts in the DynamoDBLocal artifact. An integration test version of
these sames tests will be added instead.

Change-Id: I4d8cc216d53984d7cadcb558a0cb02a3aa4e51ca
---
 hadoop-tools/hadoop-aws/pom.xml |   11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/hadoop-tools/hadoop-aws/pom.xml b/hadoop-tools/hadoop-aws/pom.xml
index ab8b5f2..56aa2fb 100644
--- a/hadoop-tools/hadoop-aws/pom.xml
+++ b/hadoop-tools/hadoop-aws/pom.xml
@@ -354,6 +354,9 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
           <forkedProcessTimeoutInSeconds>3600</forkedProcessTimeoutInSeconds>
+          <excludes>
+            <exclude>**/TestDynamoDBMetadataStore.java</exclude>
+          </excludes>
         </configuration>
       </plugin>
       <plugin>
@@ -395,14 +398,6 @@
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
-          <testExcludes>
-            <testExclude>**/TestDynamoDBMetadataStore.java</testExclude>
-          </testExcludes>
-        </configuration>
-      </plugin>
-      <plugin>
         <artifactId>maven-dependency-plugin</artifactId>
         <executions>
           <execution>
-- 
1.7.9.5

